rs/bitcoin/ckbtc/mainnet/ckbtc_archive_upgrade_2024_08_23.md

```diff
@@ -0,0 +1,52 @@
+# Proposal to upgrade the ckBTC archive canister
+
+Git hash: `3d0b3f10417fc6708e8b5d844a0bac5e86f3e17d`
+
+New compressed Wasm hash: `5c595c2adc7f6d9971298fee2fa666929711e73341192ab70804c783a0eee03f`
+
+Target canister: `nbsys-saaaa-aaaar-qaaga-cai`
+
+Previous ckBTC archive proposal: https://dashboard.internetcomputer.org/proposal/125589
+
+---
+
+## Motivation
+Upgrade the ckBTC archive canister to the latest version to add support for the [ICRC-21: Canister Call Consent Messages](https://github.com/dfinity/wg-identity-authentication/blob/fd846030109710cab67d9381485a73db424f2b07/topics/ICRC-21/icrc_21_consent_msg.md) standard.
+
+
+## Upgrade args
+
+```
+git fetch
+git checkout 3d0b3f10417fc6708e8b5d844a0bac5e86f3e17d
+cd rs/rosetta-api/icrc1/archive
+didc encode '()'
```

Comment:
For all 3 proposals, this is `4449444c0000`

Reply:
Not just for these 3 proposals.   :wink: